index.js

```diff
@@ -1,445 +0,0 @@
-    ===
-    === =
-    //unlocks final room so player can escape to safety
-    function unlock(word) {
-      >>>
-      >>> > 94 cb5f87c00a9964209b9e74dc9704ffd8edf4fa
-      if (player.inventory.includes('key')) {
-        currentState.locked = false
-        console.log("You unlocked the padlock! Now quickly make your way into the tunnel and make your way to Paradise Cove")
-      } else {
-        console.log("You don't have a key to do that")
-
-      }
-    }
-
-    //allows plater to examine item
-    function examine(word) {
-      let lookAt = itemLookUp[word]
-      if (player.location.inv.includes(word)) {
-        console.log(lookAt.desc)
-      }
-    }
-
-
-    /*----------------------------Player-------------------------------*/
-    //player state 
-    const player = {
-      inventory: [],
-      location: null,
-      encroacment: 20,
-    };
-
-    /*-------------------------Constants----------------------------------*/
-    //word bank that triggers action functions
-    const actions = {
-      move: ["move", "go", "walk", "run"],
-      consume: ["eat", "consume", "use", "wear"],
-      grab: ["grab", "take"],
-      unlock: ["unlock"],
-      examine: ["read", "examine"],
-    };
-
-    /*----------------------------------------Items---------------------------------------------------------------*/
-    //class constructor
-    class Item {
-      constructor(name, desc, takable, action) {
-        this.name = name;
-        this.desc = desc;
-        this.takable = takable;
-        this.action = action;
-      }
-    }
-
-    //items class
-    const note = new Item(
-      "Note",
-      'Hand written note that is attached the entry gate that reads ,"We have safety and supplies beyond the town through the tunnel, there is a key in one of the houses that will let get through the door at the end of the cul-de-sac and into the tunnel and make your way to Paradise Cove and make sure to throw the key into the corner for others to find and re-lock hatch the "',
-      false)
-
-    const backpack = new Item(
-      "Backpack",
-      "A weathered but usable backpack blue",
-      true);
-
-    const key = new Item("Key",
-      "a standard padlock key",
-      true);
-
-    const flashlight = new Item(
-      "Flashlight",
-      "A dim but functional flashlight. Just bright enough to lead the way!",
-      true,
-    )
-
-    const runningShoes = new Item(
-      "Running shoes",
-      "A pair of blue PF Flyers, They're guaranteed to make you run faster and job higher!",
-      true,
-    )
-
-    const rug = new Item(
-      "Rug",
-      "An out of place cheap looking throw rug about 3x6 in size.",
-      true,
-    )
-
-    const padlock = new Item(
-      "Padlock",
-      "Just a standard pad lock.",
-      true
-    )
-
-    //lookup table
-    let itemLookUp = {
-      'note': note,
-      'backpack': backpack,
-      'key': key,
-      'flashlight': flashlight,
-      'runningShoes': runningShoes,
```

Comment:
The string 'runningShoes' here will throw an error because your user input is being set .toLowerCase() when you sanitize it. This will cause the user input for the shoes to never match and not be an item that can be taken